data/presets/presets.json

```diff
@@ -10712,7 +10713,7 @@
         "fields": [
             "ref",
             "operator",
-            "network"
+            "network_foot"
```

Comment:
This seems like a mistake for `route=bus`
I'll check out the code and rerun `make` to see if maybe that fixes it...